### PR TITLE
Calendar Component

### PR DESCRIPTION
**Pull request by ameentalahmeh**
- create Calendar component

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,17 +1,23 @@
 import React, { Component } from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
 import "./App.css";
-import HomePage from "./components/HomePage";
-import Hall from "./components/Hall";
-import Header from './components/Header/header';
+
+import Calendar from "./components/Calendar";
+// import HomePage from "./components/HomePage";
```
Comment (shahenazmonia) [inline client/src/App.js]: Delete unrequired and commented lines

File: client/src/components/Calendar/index.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+         note: event.note,
+         orgName: event.org_name,
+         userId : event.eventId}
+       })
+       this.setState({
+         events,
+         loading:true
+       })
+     }).catch((error)=>{
+
```
Comment (shahenazmonia) [inline client/src/components/Calendar/index.js]: You have to handle the error that you may receive it during fetching data

File: client/src/App.js (hunk 1 above)
Comment (ameentalahmeh) [reply client/src/App.js]: done :+1:

File: client/src/components/Calendar/index.js (hunk 2 above)
Comment (ameentalahmeh) [reply client/src/components/Calendar/index.js]: done :+1:

**Pull request merged**